src/main/java/com/softserveinc/ita/pageobjects/mkaralash/GoogleSearchResultPage.java

```diff
@@ -0,0 +1,26 @@
+package com.softserveinc.ita.pageobjects.mkaralash;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static org.openqa.selenium.By.xpath;
+
+public class GoogleSearchResultPage {
```

Comment:
create an abstract BasePage class and place WebDriver variable there, also create a constructor in base page and every pageobject should extend BasePage and call the super constructor